stages/org.osbuild.chown

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Change file owner and group
+Change the file user and/or group ownership of one or more files or directories inside the tree.
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util.path import in_tree
+
+SCHEMA = r"""
+"additionalProperties": false,
+"required": ["paths"],
+"properties": {
+  "paths": {
+    "type": "array",
+    "description": "Paths to operate on for changing ownership",
+    "items": {
+      "type": "string",
+      "pattern": "^\\/(?!\\.\\.)((?!\\/\\.\\.\\/).)+$"
+    }
+  },
+  "user": {
+    "type": "string"
+  },
+  "group": {
+    "type": "string"
+  },
+  "recursive": {
+    "type": "boolean"
+  }
+}
+"""
+
+def main(tree, options):
+    paths = options["paths"]
+    user = options.get("user", "")
+    group = options.get("group", "")
+    recursive = options.get("recursive", False)
+
+    realpaths = list()
+    for p in paths:
+        realpath = os.path.join(tree, p.lstrip("/"))
+        if not in_tree(realpath, tree, must_exist=True):
+            print(f"Error: path {p} not in tree")
+            return 1
+        realpaths.append(realpath)
+
+    arguments = ["--", f"{user}:{group}"]
```

Comment:
I wonder whether we should always add the `:` to the argument even if the `group` is not provided. This way the group will be always changed to the group of the user specified before `:`. This way there would be no way to change only the user ownership of a file or a directory. One can always specify both, the `user` and `group` if they need to set group to the user's group.

Reply:
Interesting point yeah.  I think you're right: only add the `:` when a group is specified to avoid changing it to the user's default when nothing is specified.

Reply:
> I wonder whether we should always add the : to the argument even if the group is not provided
> 
>---
> I think you're right: only add the : when a group is specified to avoid changing it to the user's default when nothing is specified.

I am a bit confused here :smile:  So what do we prefere here? :)

Reply:
if only `user` is provided, then there should be no `:` after it. It should be added to the arguments only if also `group` is provided. Note that specifying only `group` is valid and in such case, the argument passed to `chown` would be `:<group>`